lib/ansible/modules/utilities/logic/wait_for.py

```diff
@@ -454,6 +462,8 @@ def _create_connection(host, port, connect_timeout):
         connect_socket.connect((host, port))
     else:
         connect_socket = socket.create_connection((host, port), connect_timeout)
+    if send_string is not None:
+        connect_socket.sendall(bytes(send_string))
```

Comment:
Use our `to_bytes()` function.
```suggestion
        connect_socket.sendall(to_bytes(send_string))
```